data/spells/lib/spells.lua

```diff
@@ -296,3 +296,57 @@ function Creature:addDamageCondition(target, conditionType, listType, damage, ti
 	target:addCondition(condition)
 	return true
 end
+
+function Creature:addPartyCondition(combat, variant, condition, baseMana)
+	local position = self:getPosition()
```

Comment:
Throughout the function, you will never need this variable more than once. You can replace all instances of `position` with `self:getPosition()` and save 1 variable.